route/evermist_island/intro.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
                 SeqTurboMashUntilIdle(name="Wait for control"),
                 SeqLog(name="SYSTEM", text="We have control!"),
                 # TODO: Need to be able to do special ability (mash fails for Zale)
```

Comment:
I believe this is still the case, it will fail for Zale?

Reply:
Until i add the timing ability for zale he has no way to do it. Its on a different skill index right now. Ill work on that next